src/pudl/metadata/resources/eia.py

```diff
@@ -19,7 +19,21 @@
     "boilers_entity_eia": {
         "description": "Static boiler attributes compiled from the EIA-860 and EIA-923 data.",
         "schema": {
-            "fields": ["plant_id_eia", "boiler_id", "prime_mover_code"],
+            "fields": [
+                "plant_id_eia",
+                "boiler_id",
+                "boiler_operating_date",
+                "boiler_manufacturer",
+                "boiler_manufacturer_code",
+                "firing_rate_using_coal_tons_per_hour",
+                "firing_rate_using_oil_bbls_per_hour",
+                "firing_rate_using_gas_mcf_per_hour",
+                "firing_rate_using_other_fuels",
+                "waste_heat_input_mmbtu_per_hour",
+                "wet_dry_bottom",
+                "fly_ash_reinjection",
```

Comment:
**Move to annual:**

Operating date: 
482 boilers from 306 plants have more than one date reported. 
- There's probably some mix of junk data and real changes here. 
- For 181 of these records, the changes in operating date that occur are changes to _planned_ operating dates (i.e., the operating date is ahead of the reporting date). Otherwise, anecdotally, these changes seem to be between sets of dates that far predate the reporting period (e.g. 1964 to 1979).
- 192 boilers have a date that changes more than a year from the first reported record to the last. (E.g., approximately 40% of these changes are persistent and substantial).
- 31 boilers have first and last dates which are identical, but one or more dates in the middle changes.
- For some very small proportion of boilers, the change in operating date is accompanied by a change in other design characteristics: 9 boilers (change in coal firing rate), 6 (change in oil firing rate), 33 (change in gas firing rate).

Hypotheses for changes: 1) plants are changing and/or mixing up the boiler IDs to which they assign equipment (e.g. renaming boiler 1 as boiler 3); 2) plants are reporting a suspected operating date for a boiler not yet in service, which is changing; 3) because the question is phrased as "in-service date", there may be a lack of clarity about whether to report the year the boiler first began operating, or the year it resumed operating after going offline for service, and practices may have changed over time; 4) plants may be substantially replacing or upgrading boilers.

Wet/dry bottoms:
Wet/dry bottom conversions are technically possible. We see this in 70 boilers, and value changes persist for multiple years.

Fly ash reinjection:
The vast majority of conversions here occur in concert with changes in other columns and persist (firing rate, turndown ratio, etc.), suggesting these reflect real design/operational changes in the boiler (or changes in what boiler is associated with the boiler ID, at least).

Firing rate changes:
As above, these seem to generally correspond to changes in other operational columns (e.g. turndown ratio).

Manufacturer / manufacturer code:
- 12 boilers from 4 plants change. 7 of these changes are from Alstom to "other, specify in schedule 7", two are from "General Electric" to "Nooter Erickson" in the same plant. All of the changes occur between 2009 and 2010. Arguably, this is the only column with a clear argument for keeping in static, but if it is the only one then perhaps it's worth just keeping all cols in the `boilers_eia860` table.

Reply:
In my experience when a boiler or generator is replaced or converted to a new fuel (like coal steam generators getting repowered with gas) they **usually** change the ID, but it seems like there no universal patterns in their reporting, so I wouldn't be surprised if there are real exceptions.

* `planned_operating_date` seems like it could be quite changeable due to construction delays, and would make sense to keep in the annual table.
* Actual `operating_date` seems like it really should be constant, notwithstanding boilers that get replaced but keep an old ID.  I suspect that having multiple `operating_date` values on a single piece of infrastructure could cause some problems downstream, where we're using (generator) `operating_date` as a feature in the record linkage connecting FERC and EIA data. @cmgosnell what do you think?

For the physically descriptive attributes like wet/dry bottoms, fly ash reinjection, firing rate changes, I think if there's ambiguity, it's probably safest to make it a (very slowly changing) annual attribute, so we're preserving the reported information.

Not sure what to do about manufacturer. Like operating data, that really seems like it should be a fixed attribute conceptually, and 12 boilers out of... thousands? Tens of thousands? Not much to lose there.

Reply:
There is no `planned_operating_date` column for boiler data - the survey asks for "actual or proposed operating date" in one entry, so these are reported in the same column. If we wanted to split them out we would have to compare this against the reporting date and make a new column here.

I will say that the majority of changes in date (~300) are multi-year changes (e.g. a boiler will report 1964 from 2009-2013, and 1976 from 2014-present). All the hypotheses I've been able to come up with for why this would happen suggest that boiler ID is not a universally reliable attribute connecting boilers within all plants across years, but perhaps I'm missing something. 

Options:
1) Move everything to annual table.
2) Only keep manufacturer in the static table.
3) Keep manufacturer and operating date in the static table, creating about 300-400 NA records for operating date
4) Make a new planned operating date column comparing against the record date, keep that in the annual table and manufacturer and operating date in the static table, creating about 200-300 NA records for operating date.

Reply:
Arguments for leaving boiler manufacturer data in static make sense to me, so let's go for option 2 here.